contracts/builder/SingleRandomWinnerPrizePoolBuilder.sol

```diff
@@ -20,11 +21,14 @@ contract SingleRandomWinnerPrizePoolBuilder is Initializable {
 
   function initialize (
     PrizePoolBuilder _prizePoolBuilder,
+    RNGInterface _rng,
     SingleRandomWinnerPrizeStrategyFactory _prizeStrategyFactory
   ) public initializer {
     require(address(_prizePoolBuilder) != address(0), "prize pool builder must be defined");
+    require(address(_rng) != address(0), "rng cannot be zero");
```

Comment:
The require statements should be consistent, would recommend "rng must be defined"